app/jobs/handover_follow_up_job.rb

```diff
@@ -0,0 +1,32 @@
+class HandoverFollowUpJob < ApplicationJob
+  queue_as :default
+
+  def perform(date)
+    ldus = LocalDivisionalUnit.all.where.not(email_address: nil)
+
+    ldus.each do |ldu|
+      offenders = ldu.teams.map { |team| team.case_information.where(com_name: nil).map(&:nomis_offender_id) }.flatten
+                  .map { |off_id| OffenderService.get_offender(off_id) }
+                  .select { |o| o.handover_start_date.present? && o.handover_start_date == date - 1.week }
```

Comment:
It might be better to use >= 1.week here to be defensive.

Reply:
We only send this once when a week has past, your suggestion means it would potentially happen every day